.circleci/config.yml

```diff
@@ -5,6 +5,11 @@ orbs:
   gcp-gcr: circleci/gcp-gcr@0.13.0
   docker: circleci/docker@1.5
 
+executors:
+  custom-ubuntu-executor:
```

Comment:
Maybe call this `ubuntu-machine-executor` since the `machine` part is significant to what this is doing for us.